_posts/2023-02-22-cockpit-286.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Cockpit 286, Cockpit-machines 284, Cockpit-podman 63
```

Comment:
Title looks a bit long in the browser...

![image](https://user-images.githubusercontent.com/10246/220666939-4c189d38-bff9-426b-826e-0ca06b25d2f0.png)

![image](https://user-images.githubusercontent.com/10246/220667033-5d1868de-95cc-43b9-88c6-218346541a0a.png)

Perhaps we should drop Cockpit- from Machines and Podman?

```suggestion
title: Cockpit 286, Machines 284, and Podman 63
```

Reply:
But then it does look like we're announcing Podman 63. Hm.

Reply:
We usually announce the Cockpit release in the title and the others at the top and in the links at the bottom.

Reply:
Changed.